sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/filter/ContainerRequestContextHolder.java

```diff
@@ -55,23 +52,21 @@ public static Runnable transferRequestContext(Runnable runnable) {
     Map<String, String> contextMap = MDC.getCopyOfContextMap();
     ContainerRequestContext requestContext = currentRequestContext().orElse(null);
 
-    return new TracedRunnable(
-        () -> {
-          ContainerRequestContextHolder containerRequestContextHolder =
-              new ContainerRequestContextHolder();
-          try {
-            if (contextMap != null) {
-              MDC.setContextMap(contextMap);
-            }
-            containerRequestContextHolder.filter(requestContext);
+    return () -> {
```

Comment:
Shouldn't this be wrapped with `Context.wrap` (or similar) now?

Reply:
Fixed on https://github.com/SDA-SE/sda-dropwizard-commons/pull/2040